scripts/check_blocking_packages.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import argparse
+
+import rosdistro
+import rosdistro.dependency_walker
+
+index = rosdistro.get_index(rosdistro.get_index_url())
+valid_distro_keys = index.distributions.keys()
+valid_distro_keys.sort()
+
+parser = argparse.ArgumentParser(description='')
+parser.add_argument('--distro', metavar='distribution', type=str,
+                    help='The distribution to check packages for',
+                    default=valid_distro_keys[-1])
+
+# If not specified, check for all repositories in the previous distribution
+parser.add_argument('--repositories',
+                    metavar='repositories', type=str, nargs='*',
```

Comment:
For the metavar I think something like `REPOSITORY_NAME` would be more helpful.

I would drop arguments which don't change the default: `type=str`